### ISSUE TITLE
Migrate to Vanilla Ore Tags

### ISSUE THREAD

**Issue by shartte** [AppliedEnergistics/Applied-Energistics-2#5281]
I just saw Vanilla adds its own tags for Vanilla ores in 1.17:

`Added the coal_ores, copper_ores, diamond_ores, emerald_ores, iron_ores, lapis_ores, and redstone_ores block and item tags. `

We should migrate to those tags in 1.17 away from the Forge tags.

**Comment by yueh** [comment]
These might be mostly for the deepslate variants, I would guess forge will still keep their own tags around for other metals and just have them include the vanilla ones.

So we might run into mods adding their own copper, but only tagging it as `forge:ores/copper` and not the vanilla tag.
Also a bit messy when we have mixed vanilla and forge tags and then have to constantly look up which one does exist.

Really something we have to wait on Forge to make a decision.

**Comment by shartte** [comment]
Agreed, we have to wait. 
We're not adding our own ores. IF forge keeps their own tags and just includes #minecraft:iron_ores, we'll keep using Forge's tags in our recipes. Otherwise we'll have to switch to vanilla tags. 

Just keep this issue as a reminder.

**Comment by Technici4n** [comment]
The standard for me (Fabric) is to define a tag in the shared namespace `forge:ingots/xxx` or `c:xxx_ingots` that inherits values from minecraft tags.

**Comment by yueh** [comment]
That is fine, I just don't want to have someone jump immediately on it. I'll move it to the backlog.

**Comment by shartte** [comment]
We have to check this as soon as a real Forge version is out primarily for grindstone integration.

**Comment by yueh** [comment]
Certainly, I just won't expect to have a definitive answer the second 1.17 is released. If they even make a decision about it now and now mark it as low priority and take months for it.

Really just in case Forge finds out in 2 days that they have to rewrite FG again, take months for it, and we decide to start API breaks and whatever else we can already prepare on 1.16 and someone jumps on this change immediately in some way.

**Comment by Technici4n** [comment]
There is no reason for forge to drop its own tags in case mods add things like space iron or nether iron. Also, for consistency.
